pinot-common/src/main/java/com/linkedin/pinot/common/utils/CommonConstants.java

```diff
@@ -312,6 +312,7 @@ public ServerType getServerType() {
     public static final String CRC = "segment.crc";
     public static final String CREATION_TIME = "segment.creation.time";
     public static final String FLUSH_THRESHOLD_SIZE = "segment.flush.threshold.size";
+    public static final String FLUSH_THRESHOLD_TIME_MILLIS = "segment.flush.threshold.time.millis";
```

Comment:
can we make it a string of "segment.flush.threshold.time", and format it exactly like the flush threshold time in config? (i.e. be able to accept something like "6h")

Reply:
Changed it to use period string. Have kept the getter and setter of the timeThresholdToFlushSegment to operate on millis, and only internally handled it as Period. I did this to avoid having to make every caller of the setter and getter to create their own period string using their own period formatter.